disnake/message.py

```diff
@@ -119,9 +115,7 @@ def convert_emoji_reaction(emoji: Union[EmojiInputType, Reaction]) -> str:
         # No existing emojis start/end with `<>` or `:`, so this should be okay.
 
         s = emoji.strip("<>:")
-        # `str.removeprefix` is py 3.9 only
-        if s.startswith("a:"):
-            s = s[2:]
+        s = s.removeprefix("a:")
         return s
```

Comment:
this triggers [RET504](https://docs.astral.sh/ruff/rules/unnecessary-assign)
```suggestion
        return emoji.strip("<>:").removeprefix("a:")
```